AgateDecorator/AgateDecorator.js

```diff
@@ -1,9 +1,9 @@
 /**
- * Applies Agate specific styling and behaviors.
+ * {@link agate/AgateDecorator} will be removed in 2.0.0. Use {@link agate/ThemeDecorator} instead.
```

Comment:
There are another broken links. (not this Component)

agate/ColorPicker
agate/ContextualPopupDecorator/
agate/Input
agate/Panels
....

Please search "@see" in the agate

Reply:
That's good point, I found the probluem It's not only agate but also all other UI frameworks. 
On local server is linking but I cannot see the link on the web server. Currently I don't know what is different between them. So, how about fix them another ticket?

exmaple) ColorPicker `* @see {@link ui/Changeable.Changeable} ` on local
```
<div class="--see">
 ::before
  "See: "
  <span>
    <a data-tooltip="ui/Chanageable" href="/docs/modules/ui/Changeable/@Changeable">ui/Changeable.Changeable</a>
  </span>
</div>
```
whereas on jenkins server
```
<div class="--see">
 ::before
  "See: "
  ::after
</div>
```